client/rest-high-level/build.gradle

```diff
@@ -34,14 +34,32 @@ publishing {
     }
 }
 
+// We bundle the x-pack:protocol project into the jar.
+configurations {
+    bundled
```

Comment:
Whoops, another spot where indentation doesn't match the rest of the project. I'll fix.